packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -140,8 +138,8 @@ export const ResizableBox = ({
     const [newWidth, setNewWidth] = useState<number>(width || minWidth);
     const [newHeight, setNewHeight] = useState<number>(height || minHeight);
     const [isResizing, setIsResizing] = useState<boolean>(false);
-    const [isHovering, setIsHovering] = useState(false);
-    const [direction, setDirection] = useState<Directions>();
+    const [isHovering, setIsHovering] = useState<boolean>(false);
+    const [direction, setDirection] = useState<Directions>(null);
```

Comment:
Please use it this way:

```
type Directions = 'top' | 'left' | 'right' | 'bottom';

...

const [direction, setDirection] = useState<Directions | null>(null);
```

Reply:
I tried exactly this one but from some reasons, it was not possible.. Will add more details in a moment..

Reply:
There was some another typo, already fixed!